test/helper/test_trigger.ts

```diff
@@ -172,8 +172,24 @@ const testTriggerFile = (file: string) => {
 
       let captures = 0;
       const currentNetRegex = currentTrigger.netRegex;
+
       if (currentNetRegex !== undefined && currentNetRegex !== null) {
-        const capture = new RegExp(`(?:${currentNetRegex.toString()})?`).exec('');
+        let netRegexRegex: RegExp;
```

Comment:
Can you submit these code changes separately so that this PR is just replacing netRegex with parameters?